### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by ligoore** [dannyfritz/atom-date#3]
Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:
```json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```
```
Package.getActivationCommands (C:\Users\Mateusz\AppData\Local\atom\app-0.201.0\resources\app.asar\src\package.js:808:9)
Package.hasActivationCommands (C:\Users\Mateusz\AppData\Local\atom\app-0.201.0\resources\app.asar\src\package.js:733:20)
<unknown> (C:\Users\Mateusz\AppData\Local\atom\app-0.201.0\resources\app.asar\src\package.js:185:24)
Package.measure (C:\Users\Mateusz\AppData\Local\atom\app-0.201.0\resources\app.asar\src\package.js:163:15)
Package.load (C:\Users\Mateusz\AppData\Local\atom\app-0.201.0\resources\app.asar\src\package.js:177:12)
PackageManager.loadPackage (C:\Users\Mateusz\AppData\Local\atom\app-0.201.0\resources\app.asar\src\package-manager.js:355:14)
```

**Comment by dotnetCarpenter** [comment]
According to the current release notes, `atom-date` should not load. But it does... We'll see if it completely fails on next week atom update or if it is an empty threat.

**Comment by jason-o-matic** [comment]
I'm still seeing this deprecation warning.

**Comment by jschrab** [comment]
I am still seeing this warning too.

**Comment by dannyfritz** [comment]
I'm not seeing the deprecation error. Is everyone still seeing it and wanting me to go ahead and migrating to the newer API?

**Comment by jason-o-matic** [comment]
Now that you mention it the deprecation warning seems to have gone away.

**Comment by dannyfritz** [comment]
Strange, I don't see any mention of `activationEvents` in the 1.0.x docs. Therfore, I think I'll go ahead with migrating to `activationCommands`. Just asking for some quick advice in the #atom IRC channel first.

**Comment by dannyfritz** [comment]
I saw the deprecation errors again. No idea what was going on there. Fixed it in 3a8311a63a690e3b9d611c0e6fa4e45d2e0f5cd5.
